client/me/purchases/purchases-list/index.jsx

```diff
@@ -177,7 +177,12 @@ class PurchasesList extends Component {
 				<PageViewTracker path="/me/purchases" title="Purchases" />
 				<MeSidebarNavigation />
 
-				<FormattedHeader brandFont headerText={ titles.sectionTitle } align="left" />
+				<FormattedHeader
+					brandFont
+					headerText={ titles.sectionTitle }
+					subHeaderText={ translate( 'View, manage, or cancel your plan and other purchases.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'View, manage, or cancel your WordPress.com plan and other purchases.' )` *ES Score: 10*

Reply:
Resolving since this is a new string being added intentionally.